elements/lisk-validator/src/errors.ts

```diff
@@ -31,11 +30,85 @@ export interface ErrorObject {
 	data?: never;
 }
 
+interface ErrorParams {
+	[key: string]: unknown;
+}
+
+const errorParamToString = (
+	param: string | Buffer | BigInt | undefined | unknown,
+): string => {
+	let paramAsString = '';
+	if (typeof param === 'bigint') {
+		paramAsString = param.toString();
+	} else if (Buffer.isBuffer(param)) {
+		paramAsString = param.toString('base64');
+	} else if (param === undefined) {
+		paramAsString = '';
+	} else {
+		paramAsString = param as string;
+	}
+	return paramAsString;
+};
+
+const errorFormatter = (error: ErrorObject): string => {
+	let errorMessage = '';
+	switch (error.keyword) {
+		case 'type':
+			errorMessage = `Property '${
+				error.dataPath ?? ''
+			}' should be of type '${errorParamToString(error.params.type)}'`;
+			break;
+		case 'additionalProperties':
+			errorMessage = `Property '${
+				error.dataPath ?? ''
+			}' has extraneous property '${errorParamToString(
+				error.params.additionalProperty,
+			)}'`;
+			break;
+		case 'minLength':
+			errorMessage = `Property '${error.dataPath ?? ''}' ${errorParamToString(
+				error.message,
+			)}`;
+			break;
+		case 'maxLength':
+			errorMessage = `Property '${error.dataPath ?? ''}' ${errorParamToString(
+				error.message,
+			)}`;
+			break;
+		case 'format':
+			errorMessage = `Property '${error.dataPath ?? ''}' ${errorParamToString(
+				error.message,
+			)}`;
+			break;
+		case 'required':
+			errorMessage = `Missing property, ${errorParamToString(error.message)}`;
+			break;
+		case 'dataType':
+			errorMessage = `Property '${error.dataPath ?? ''}' ${errorParamToString(
+				error.message,
+			)}`;
+			break;
+		default:
+			errorMessage = error.message ?? 'unspecified validator error';
```

Comment:
Use capitalize words as in other messages. 

```suggestion
			errorMessage = error.message ?? 'Unspecified validator error';
```